### PR TITLE
Introduce resource as filename, add more details to GUI

### PR DESCRIPTION
**Pull request by datazuul**
- former filename was the UUID.extension which is very long and not necessary (UUID is reflected by the path). now the filename is "resource".extension. Migration for existing database entries and files added, too.

- in this context it was helpful to see current path/uri/storage in GUI, added mor details to fileresource-view-page

- find method (not used yet) was implemented wrong (fixed it) and additionally has to be changed for new filename, too.

### CODE REVIEW

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/test/java/de/digitalcollections/cudami/server/backend/impl/database/migration/V1_6_3__DML_Change_fileresource_filename_to_resourceTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class V1_6_3__DML_Change_fileresource_filename_to_resourceTest {
+
+  @Test
+  public void testConvertUri() throws Exception {
+    System.out.println("convertUri");
```
Comment (morpheus-87) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/src/test/java/de/digitalcollections/cudami/server/backend/impl/database/migration/V1_6_3__DML_Change_fileresource_filename_to_resourceTest.java:11]: Braucht es diese Ausgabe tatsächlich?

File: dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml [hunk 2]
```diff
@@ -73,6 +73,21 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
```
Comment (morpheus-87) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml:88]: Wofür genau braucht es diese Abhängigkeit?

File: dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml [hunk 3]
```diff
@@ -73,6 +73,21 @@
       <groupId>org.json</groupId>
       <artifactId>json</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
```
Comment (morpheus-87) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml:77]: Hier könnte das Meta-Package von JUnit-Jupiter verwendet werden, wie [hier](https://github.com/dbmdz/cudami/blob/introduce-resource-as-filename/dc-cudami-server/dc-cudami-server-backend-file/pom.xml#L26).

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/test/java/de/digitalcollections/cudami/server/backend/impl/database/migration/V1_6_3__DML_Change_fileresource_filename_to_resourceTest.java (hunk 1 above)
Comment (datazuul) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/src/test/java/de/digitalcollections/cudami/server/backend/impl/database/migration/V1_6_3__DML_Change_fileresource_filename_to_resourceTest.java:11]: brauchen nicht, ..., ich hab es gelsöcht

File: dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml (hunk 3 above)
Comment (datazuul) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml:77]: done

File: dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml (hunk 2 above)
Comment (datazuul) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/pom.xml]: entfernt

File: dc-cudami-server/dc-cudami-server-webapp/pom.xml [hunk 4]
```diff
@@ -193,9 +193,13 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <executions>
           <execution>
+            <id>repackage</id>
             <goals>
               <goal>repackage</goal>
             </goals>
+            <configuration>
```
Comment (morpheus-87) [inline dc-cudami-server/dc-cudami-server-webapp/pom.xml:200]: Wird diese Änderung tatsächlich benötigt? Bisher hatten wir auch keine Probleme.

**Pull request merged**